viewing_party/main.py

```diff
@@ -0,0 +1,193 @@
+# WAVE ONE  
+
+def create_movie(title, genre, rating):
+    movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title == None or genre == None or rating == None:
+        return None 
+    return movie_dict
+
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data 
+
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            return user_data
+    return user_data
+
+
+
+# WAVE TWO 
+
+    # PROBLEM ONE 
+    # I need to calculate the total sum of the ratings
+    # Then I need to divide the total sum by the length of the movies in the watched list 
+def get_watched_avg_rating(user_data):
+    sum_counter = 0
+    for movie in user_data["watched"]:
+        sum_counter += movie["rating"] 
+    if len(user_data["watched"]) == 0:
+        return 0.0 
+
+    average_rating = sum_counter/len(user_data["watched"])
+    return average_rating 
+
+    # PROBLEM TWO 
+    # iterating through the watched list
+    # each loop gives us one movie 
+    # get genre from movie 
+    # add new genre into the dictionary as a key with count being the value, (check if genre is in the dict and add one) OR (add too dict at 1)
+    # Now iterate over different_genre dict
+    # Find highest value  
+    # return key to highest value 
+def get_most_watched_genre(user_data):
+    if not user_data["watched"]:
+        return None
+
+    genres = {}
+    for movie in user_data["watched"]:
+        for i, x in movie.items():
+            if i == "genre":
+                if x in genres:
+                    genres[x] += 1 
+                else:
+                    genres[x] = 1
```

Comment:
You don't need to loop through a dictionary if you already know the key you want.  In addition, variable names like x are ok for counter variables, but try to provide more meaningful variable names for other use cases. See the code suggestion below.

```suggestion
   for movie in user_data["watched"]:
        genre = movie["genre"]
        if genre in genres:
                genres[genre] += 1 
        else:
                genres[genre] = 1
```